hack/ginkgo-e2e.sh

```diff
@@ -144,6 +144,8 @@ if [[ "${GINKGO_NO_COLOR}" == "y" ]]; then
   ginkgo_args+=("--noColor")
 fi
 
+CONTAINER_RUNTIME=${CONTAINER_RUNTIME:-${KUBE_CONTAINER_RUNTIME:-}}
```

Comment:
there are 2 code paths in 2 different CI jobs, So we have to check if either variable is set then we pick it up.